Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js

```diff
@@ -528,8 +524,11 @@ WI.CSSManager = class CSSManager extends WI.Object
 
     // PageObserver
 
+    // COMPATIBILITY (macOS 13, iOS 16.0): `Page.defaultAppearanceDidChange` was removed in favor of `Page.defaultUserPreferencesDidChange`
     defaultAppearanceDidChange(protocolName)
     {
+        console.assert(InspectorBackend.Enum.Page.Appearance, "Legacy protocol: Appearance type missing from Page domain.", protocolName);
```

Comment:
I'm not sure the assertion message is needed here - the entire method is marked with `COMPATIBILITY ...`, and the missing type is inferable from the condition.

Also seems odd to include the `protocolName` parameter in this assertion since it's not directly related to the condition being tested. (If you are trying to also assert that there is a `protocolName`, it would be clearer to make it a separate assertion IMO.)

Reply:
Ok. Removed the assertion altogether since this method is only called by `WI.PageObserver`, its sole intended caller.